packit/config/aliases.py

```diff
@@ -157,7 +157,7 @@ def get_branches(*name: str, default=DEFAULT_VERSION) -> Set[str]:
 
     for sys_and_version in get_versions(*names):
         if "rawhide" in sys_and_version:
-            branches.add("master")
+            branches.add("main")
```

Comment:
This change doesn't seem to be done in `src.fedoraproject.org` yet.

Reply:
That's right. I've just made it more generic.